pkg/api/types.go

```diff
@@ -2035,36 +2053,35 @@ type ComponentStatusList struct {
 // will be populated based on the Container configuration if they are not set.  Defining them on
 // both the Container AND the SecurityContext will result in an error.
 type SecurityContext struct {
-	// Capabilities are the capabilities to add/drop when running the container
+	// Capabilities are the capabilities to add/drop when running containers.
+	// Optional: Defaults to the default set of capabilities granted by the container
```

Comment:
Going to hold off changing this until we recertify that we have agreement on what the semantics should be.